core/designsystem/src/main/kotlin/com/dhc/designsystem/tipcard/DhcTipCard.kt

```diff
@@ -0,0 +1,138 @@
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.layout.wrapContentSize
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.tooling.preview.PreviewParameter
+import androidx.compose.ui.tooling.preview.PreviewParameterProvider
+import androidx.compose.ui.unit.dp
+import coil3.compose.AsyncImage
+import com.dhc.designsystem.DhcTheme
+import com.dhc.designsystem.DhcTypoTokens
+import com.dhc.designsystem.LocalDhcColors
+import com.dhc.designsystem.R
+import com.dhc.designsystem.SurfaceColor
+
+
+@Composable
+fun DhcTipCard(
+    tipCardItem: TipCardModel,
+    modifier: Modifier = Modifier
+) {
+    Column(
+        modifier = modifier
+            .fillMaxWidth()
+            .background(color = SurfaceColor.neutral700, shape = RoundedCornerShape(12.dp))
+            .padding(horizontal = 16.dp, vertical = 20.dp)
+    ) {
+        TipCardTitle(
+            title = tipCardItem.title,
+            icon = tipCardItem.icon
+        )
+        Spacer(modifier = Modifier.height(8.dp))
+        TipCardContent(
+            color = tipCardItem.color,
+            content = tipCardItem.cont
+        )
+    }
+}
+
+@Composable
+fun TipCardTitle(
+    title: String,
+    icon: String
+) {
+    Row(
+        modifier = Modifier.fillMaxWidth()
+    ) {
+        AsyncImage(
+            model = icon,
+            contentDescription = "tipCardIcon",
+            modifier = Modifier.size(20.dp),
+        )
+        Spacer(modifier = Modifier.width(4.dp))
+        Text(
+            text = title,
+            style = DhcTypoTokens.Body5,
+            color = SurfaceColor.neutral400,
+            textAlign = TextAlign.Center,
+        )
+    }
+}
+
+
+@Composable
+fun TipCardContent(
+    color: String?,
+    content: String
+) {
+    val dhcColor = LocalDhcColors.current
+    Row(
+        modifier = Modifier.wrapContentSize(),
+        horizontalArrangement = Arrangement.Center,
+        verticalAlignment = Alignment.CenterVertically
+    ) {
+        color?.let {
+            Canvas(modifier = Modifier.size(8.dp)) {
+                drawCircle(
+                    color = hexToColor(color),
+                    radius = size.minDimension / 4,
+                )
```

Comment:
```suggestion
                    radius = size.minDimension / 2,
                )
```
먼가 radius 면 반지름이니까,,, size 의 4가 아니라 2를 나누어야 하는거 아니야?! 그래서 작은 것 같기두 하구..?